### PR TITLE
Improve some stuff

### PR DESCRIPTION
**Pull request by RakuDEV**
ヽ(^。^)丿

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+data/cache
```
Comment (MrIlliteracy) [inline .gitignore]: Huh?

**Comment by RedDuckss** [comment]
I have tested this pull request and it seems that your changes did not fix the downloading bug. This screenshot is taken after running this build http://i.imgur.com/mcCOcI2.png

I have also made some changes locally, which I have not yet pushed to the repo, which change a few things in the back end, one of them being a change to how the game data is downloaded, which fixes the issue (as well as adding an indicator that game data is downloading).

Additionally, I have come to find out that the API I have been using (https://thegamesdb.net/) is very spotty, and has (as of now) gone down. Due to this I am looking towards scraping a working API, saving all data (possibly including base64 versions of all images) to one file, which can be referenced locally (also removing all need for internet access).

I am currently using a public csv file containing over 3,000 games in order to server higher quality images (this change was made locally, and is not pushed to the repo. Additionally the data served are URLs to online images, which still requires internet access) and am planning on editing it to contain all required information, including the base64 images.

Because of these changes, I have decided to not merge this PR, but I would like to ask that you review the changes I am about to push as those are the most recent ones (as well as the fact that I like a few of the smaller changes you made).

**Comment by MrIlliteracy** [comment]
Some of those are good bug fixes, I'll put them in @RakuDEV

**Comment by RakuDEV** [comment]
On the first run (so without any config files) the program doesn't load anything (if it hasn't completely downloaded the image yet).
So you need to open it again.
With [this](https://github.com/RedDuckss/CemuManager/pull/3/commits/2f56ecb8dcd5803e694e4e45261818f249d877b7#diff-0364f57fbff2fabbe941ed20c328ef1aR132) commit  I've tried to fix that problem in that version.

**Comment by RedDuckss** [comment]
In light of the fact that the API used by this program had gone down around the time I tested your PR (and was possibly the cause of the error mentioned previously), I have tested it a second time now that the API is fully operational again. I have found that the commit made in regards to game loading causes a type of looping error, prompting the game folder select screen multiple times after selecting the folder before continuing on to select the Cemu folder. Due to this, and the previously mentioned un-pushed changes made by both MrIlliteracy and I, we have decided to not fully incorporate the commits made here and have not merged this PR.

However we did like nearly all your other changes and commits, and have since added those in. We thank you for the edits and commits, and we invite you to review the project again after the next update for more possible improvements.

**Comment by RakuDEV** [comment]
@RedDuckss because the "okay" button is still visible. Try [this](https://github.com/RakuDEV/CemuManager/commits/stable) branch.

**Pull request closed without merging**